src/main/java/com/artipie/rpm/meta/XmlWriterWrap.java

```diff
@@ -0,0 +1,258 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm.meta;
+
+import javax.xml.namespace.NamespaceContext;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamWriter;
+
+/**
+ * Envelop for XmlFile Class.
+ * @since 0.7
+ * @checkstyle DesignForExtensionCheck (500 lines)
+ */
+@SuppressWarnings("PMD.TooManyMethods")
+public class XmlWriterWrap implements XMLStreamWriter {
+    /**
+     * XML stream.
+     */
+    private final XMLStreamWriter xml;
+
+    /**
+     * Ctor.
+     * @param xml XMLStreamWriter object.
+     */
+    public XmlWriterWrap(final XMLStreamWriter xml) {
+        this.xml = xml;
+    }
+
+    @Override
+    public void writeStartElement(
+        final String localname
+    ) throws XMLStreamException {
+        this.xml.writeStartElement(localname);
+    }
+
+    @Override
+    public void writeAttribute(
+        final String localname,
+        final String value
+    ) throws XMLStreamException {
+        this.xml.writeAttribute(localname, value);
+    }
+
+    // @checkstyle ParameterNumberCheck (4 lines)
+    @SuppressWarnings("PMD.UseObjectForClearerAPI")
+    @Override
+    public void writeAttribute(
+        final String prefix,
+        final String namespaceuri,
+        final String localname,
+        final String value
+    ) throws XMLStreamException {
+        // Not implemeneted.
+    }
+
+    @Override
+    public void writeAttribute(
+        final String namespaceuri,
+        final String localname,
+        final String value
+    ) throws XMLStreamException {
+        // Not implemeneted.
```

Comment:
@ammaratef45 please update all the unimplemented methods, its not a big effort we just need to delegate methods.

Reply:
@Vatavuk check 5492a61